### PR TITLE
Improve news date format

### LINKED ISSUE #27: News timeago doesn't work
All news has the title "Skapad för några dagar sen".

### PR DESCRIPTION
**Pull request by klaseskilson**
Instead of using the time ago structure, it now posts the absolute date (no time).

This should close #27, so add `Fixes #27` to the merge commit message and it should close.

### CODE REVIEW

File: app/views/news/show.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<%= link_to 'Redigera', edit_news_path(@news) %>
     <%= link_to 'Tillbaka', news_index_path %>
 		<% end %>
-        <h5>Skapad <%= distance_of_time_in_words_to_now(@news.created_at) %> av <%= @news.user.name if @news.user %></h5>
+        <h5><%= l @news.created_at, format: '%e %B %Y' %> av <%= @news.user.name if @news.user %></h5>
```
Comment (tistatos) [inline app/views/news/show.html.erb]: Is it overkill to have time if the news was created today?

Comment (klaseskilson) [inline app/views/news/show.html.erb]: I don't think so. I think it would only be confusing if a user stumbles upon a news entry without a date.

Comment (klaseskilson) [inline app/views/news/show.html.erb]: Ooooh - my bad!

Comment (klaseskilson) [inline app/views/news/show.html.erb]: Yes, maybe it is a good idea to add a time stamp if the news entry was created less than 48~ hours ago.

**Comment by klaseskilson** [comment]
Recent news now have the hours printed:

screen shot 2015-12-02 at 16 45 21

**Comment by tistatos** [comment]
:+1: nice solution!

**Pull request merged**